### PR TITLE
Fix notarization rule

### PR DESCRIPTION
**Pull request by pmikolajczyk41**

### CODE REVIEW

**Review by ruslanye — APPROVED** [review]

File: src/node/dishonest_node.cpp [hunk 1]
```diff
@@ -143,7 +143,7 @@ void DishonestNode::handleVote(const Message& message, unsigned k) {
     }
     
     votes[k][epoch].insert(message.from());
-    if (3 * votes[k][epoch].size() >= 2 * numOfNodes) {
+    if (3 * votes[k][epoch].size() > 2 * numOfNodes) {
```
Comment (kostekIV) [inline src/node/dishonest_node.cpp:146]: This is good moment to add testcase when `nrOfNodes % 3 == 0`

**Review by kostekIV — CHANGES_REQUESTED** [review]
Gj one small comment

File: src/node/dishonest_node.cpp (hunk 1 above)
Comment (ruslanye) [reply src/node/dishonest_node.cpp:146]: We can create a separate issue for it.

Comment (pmikolajczyk41) [reply src/node/dishonest_node.cpp:146]: oh sorry, didn't see that and added already

Comment (kostekIV) [reply src/node/dishonest_node.cpp:146]: no need for separate issue, this test belongs here

**Review by kostekIV — APPROVED** [review]
Amazing!!!

**Pull request merged**